aggsender/query/aggchain_fep_rollup_query.go

```diff
@@ -64,6 +72,7 @@ func NewAggchainFEPQuerier(
 	logger *log.Logger,
 	aggsenderMode types.AggsenderMode,
 	aggchainFEPAddr common.Address,
+	opNodeURL string,
```

Comment:
WDYT, recieve the config instead of `aggchainFEPAddr` and `opNodeURL` (if possible due to imports)?

```suggestion
	cfg validator.FEPConfig,
```

Reply:
It can't because it is instantiated from both regular and validator aggsender which have different config structs. This is ok in my opinion, it doesn't have too much parameters in the constructor.